scripts/types.d.ts

```diff
@@ -0,0 +1,17 @@
+export type ComponentsEventHandlers = {
+  [name: string]: EventHandler[];
+};
+
+export interface EventHandler {
+  origin: string;
+  name: string;
+  description: string;
+  kind: "function";
```

Comment:
what is `kind` used for ?

Reply:
To differentiate members from event handlers.
Since this file is used just for exporting event handlers, everything is a function.